main.tf

```diff
@@ -19,6 +19,14 @@ resource "aws_vpc" "default" {
   tags                             = module.label.tags
 }
 
+resource "aws_default_security_group" "default" {
```

Comment:
```suggestion
# If `aws_default_security_group` is not defined, it would be created implicitly with access `0.0.0.0/0`
resource "aws_default_security_group" "default" {
```